tasks/release_from_changelog.rake

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'bump'
+
+desc 'Generate release notes from current changelog'
+task :release_from_changelog, [:release_type] do |_task, args|
```

Comment:
This task should probably the renamed as well.

Reply:
It still helps you create a release from the changelog… How about `create_version`? `cut_release`? Perhaps `rake version:patch`, `rake version:minor` etc?

Reply:
`cut_release` is probably the best name.